p1_data_client_python/client.py

```diff
@@ -181,7 +182,9 @@ def _parse_search(self, response: requests.Response) -> pd.DataFrame:
     def _parse_payload(response: requests.Response) -> pd.DataFrame:
         """Parse payload response and return pandas Dataframe."""
         payload_response = response.json()
-        return pd.DataFrame(payload_response["payload_data"])
+        payload = pd.DataFrame(payload_response["payload_data"])
+        payload["period"] = hdatet.to_datetime(payload["original_period"])
```

Comment:
I'd like to add here some mock tests that passes various/bad date formats to test that hdatet.to_datetime works properly. I'm ok to add it later.